README.md

```diff
@@ -1,5 +1,6 @@
 # Split
-![Split Logo](https://i.imgur.com/qXONyae.jpg)
+<img src="https://i.imgur.com/qXONyae.jpg" width="400" height="250" /> 
```

Comment:
Resizing the logo like this kinda changes the resolution and makes it look a bit squished. Would prefer if the image was cropped and reuploaded!

Reply:
Could either @SamNeale or @michellemtan look into cropping and reuploading the image Thanks

Reply:
``` suggestion
![split](https://user-images.githubusercontent.com/8885794/77110524-91a8a080-6a8a-11ea-9ad0-950b17ac6c93.png)
```
Maybe something like this. I just eyeballed it so if someone actually has the source files feel free to do it properly.